core/domain/skill_domain_test.py

```diff
@@ -71,6 +71,18 @@ def test_valid_misconception_id(self):
         self._assert_validation_error(
             'Expected misconception ID to be an integer')
 
+    def test_valid_misconception_name(self):
+        misconception_name = 'This string is smaller than 50'
+        self.skill.update_misconception_name(0, misconception_name)
+        self.skill.validate()
+        misconception_name = """This string is a larger string
+                            and it is greater than 50 chars."""
+        self.skill.update_misconception_name(0, misconception_name)
+        self._assert_validation_error(
+            """The length of misconception_name should
```

Comment:
Use single-quote (`'`). See other tests for reference (`git grep -2 _assert_validation_error` )